pipeline-templates/job/code-build.yml

```diff
@@ -1,47 +1,47 @@
 parameters:
   SolutionBaseName:
-  BuildConfiguration:
 
 jobs:
 - job: CodeBuild
+  pool:
+    name: DAS - Continuous Integration
   variables:
   - group: BUILD Management Resources
+  workspace:
+    clean: all
   steps:
-    - task: gitversion/setup@0
-      displayName: gitversion/setup
-      inputs:
-        versionSpec: 5.x
+  - task: UseGitVersion@5

```

Comment:
I believe there is a step template for it now

Reply:
I've updated this with the template step from the building block.